proto/stream_plan.proto

```diff
@@ -968,7 +968,7 @@ message StreamActor {
 
   uint32 actor_id = 1;
   uint32 fragment_id = 2;
-  StreamNode nodes = 3;
+  StreamNode nodes = 3 [deprecated = true];
```

Comment:
Is `StreamActor` still used for persistence with SQL meta store? Can we directly remove this field if not?

Reply:
Removed the field together with the previously deprecated `upstream_actor_id`. Will run backward-compat test to see if it's ok to remove it.